shared/src/packages/users/libs/enums/users-api-path.enum.ts

```diff
@@ -1,5 +1,6 @@
 const UsersApiPath = {
   ROOT: '/',
+  DELETE_USER: '/delete-user/:id',
```

Comment:
```suggestion
  $ID: '/:id',
```
I think it would be better to send DELETE request to /:id endpoint